src/Sylius/Bundle/OrderBundle/Form/DataMapper/OrderItemQuantityDataMapper.php

```diff
@@ -60,8 +60,7 @@ public function mapFormsToData($forms, &$data)
 
         $formsOtherThanQuantity = [];
         foreach ($forms as $key => $form) {
-            if ('quantity' === $form->getName()) {
-                $targetQuantity = $form->getData();
+            if ('quantity' === $form->getName() && $targetQuantity = $form->getData()) {
```

Comment:
interesting...

I'm not sure that this is right though, because it's line 73 that causes the error, not here. And this change would push *more* forms through the parent data mapper...